bin/sync_slurm_acct.py

```diff
@@ -75,6 +75,12 @@ def main():
             None,
         ),
         'start_timestamp': ('Timestamp to start the sync from', str, 'store', None),
```

Comment:
a few lines higher, also user strlist (and remov ethe split below); but also adapt the help to the new variables and explain what really is the default now

Reply:
Change creep :p